subprojects/common/analysis/src/test/kotlin/hu/bme/mit/theta/analysis/multi/ProductOfTwoTest.kt

```diff
@@ -0,0 +1,137 @@
+/*
+ *  Copyright 2024 Budapest University of Technology and Economics
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package hu.bme.mit.theta.analysis.multi
+
+import hu.bme.mit.theta.analysis.algorithm.ArgBuilder
+import hu.bme.mit.theta.analysis.algorithm.cegar.BasicAbstractor
+import hu.bme.mit.theta.analysis.algorithm.cegar.CegarChecker
+import hu.bme.mit.theta.analysis.expl.*
+import hu.bme.mit.theta.analysis.expr.refinement.*
+import hu.bme.mit.theta.analysis.stmtoptimizer.DefaultStmtOptimizer
+import hu.bme.mit.theta.analysis.unit.UnitState
+import hu.bme.mit.theta.cfa.CFA
+import hu.bme.mit.theta.cfa.analysis.CfaAction
+import hu.bme.mit.theta.cfa.analysis.CfaAnalysis.create
+import hu.bme.mit.theta.cfa.analysis.CfaPrec
+import hu.bme.mit.theta.cfa.analysis.CfaState
+import hu.bme.mit.theta.cfa.analysis.lts.CfaLts
+import hu.bme.mit.theta.cfa.analysis.lts.CfaSbeLts
+import hu.bme.mit.theta.cfa.analysis.prec.GlobalCfaPrec
+import hu.bme.mit.theta.cfa.analysis.prec.RefutationToGlobalCfaPrec
+import hu.bme.mit.theta.cfa.dsl.CfaDslManager
+import hu.bme.mit.theta.common.logging.ConsoleLogger
+import hu.bme.mit.theta.common.logging.Logger
+import hu.bme.mit.theta.core.type.booltype.BoolExprs.Not
+import hu.bme.mit.theta.core.type.booltype.BoolExprs.True
+import hu.bme.mit.theta.solver.Solver
+import hu.bme.mit.theta.solver.z3.Z3SolverFactory
+import hu.bme.mit.theta.xsts.XSTS
+import hu.bme.mit.theta.xsts.analysis.*
+import hu.bme.mit.theta.xsts.dsl.XstsDslManager
+import org.junit.Test
+import org.junit.jupiter.api.Assertions.assertTrue
+import java.io.FileInputStream
+import java.io.IOException
+import java.io.SequenceInputStream
+
+class ProductOfTwoTest {
```

Comment:
I think "MultiAnalysisTest" would be a better fit here

Reply:
Done in 55492d7